.github/workflows/benchmark-runtime-weights.yml

```diff
@@ -24,6 +31,20 @@ jobs:
   build-and-benchmark:
     runs-on: self-hosted
     steps:
+      - name: Set env
+        run: |
+          chain=""
+          if [ "${{ github.event.inputs.litmus }}" = "true" ]; then
+            chain="$chain litmus"
+          fi
+          if [ "${{ github.event.inputs.litentry }}" = "true" ]; then
+            chain="$chain litentry"
+          fi
+          if [ "${{ github.event.inputs.rococo }}" = "true" ]; then
+            chain="$chain rococo"
+          fi
+          echo "CHAIN=$chain" >> $GITHUB_ENV
```

Comment:
I believe we have to exit early if `chain` is empty -- no docker image has to be built if so

Reply:
correct it